### ISSUE TITLE
[FEA]: Make logits a constructor argument to TritonInferenceStage

### ISSUE THREAD

**Issue by dagardner-nv** [nv-morpheus/Morpheus#1378]
### Is this a new feature, an improvement, or a change to existing functionality?

Change

### How would you describe the priority of this feature request

Low (would be nice)

### Please provide a clear description of problem this feature solves

Currently this functionality is hard-coded to the NLP pipeline.

### Describe your ideal solution

For backwards compatibility, we can make this argument default to `True` for `TritonInferenceNLP`, and `False` for the other subclasses.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct
- [X] I have searched the [open feature requests](https://github.com/nv-morpheus/Morpheus/issues?q=is%3Aopen+is%3Aissue+label%3A%22feature+request%22%2Cimprovement%2Cenhancement) and have found no duplicates for this feature request
